test/unit/range/views/view_minimiser_test.cpp

```diff
@@ -73,73 +106,138 @@ class minimiser_test : public ::testing::Test
+    auto empty_view = seqan3::views::minimiser(too_short_text | kmer_view, too_short_text | rev_kmer_view, 5);
     EXPECT_TRUE(std::ranges::empty(empty_view));
-    EXPECT_RANGE_EQ(result3_ungapped_no_rev, text3 | kmer_view | minimiser_view);
+    auto empty_view2 = too_short_text | kmer_view | minimiser_no_rev_view;
+    EXPECT_TRUE(std::ranges::empty(empty_view2));
+    EXPECT_RANGE_EQ(result3_ungapped, seqan3::views::minimiser(text3 | kmer_view, text3 | rev_kmer_view, 5));
+    EXPECT_RANGE_EQ(result3_ungapped_no_rev, text3 | kmer_view | minimiser_no_rev_view);
 
 }
 
 TEST_F(minimiser_test, gapped_kmer_hash)
 {
-    EXPECT_RANGE_EQ(result1, text1 | gapped_kmer_view | minimiser_view);
-    EXPECT_THROW(text1_short | gapped_kmer_view | minimiser_view2, std::invalid_argument);
-    auto empty_view = too_short_text | gapped_kmer_view | minimiser_view;
+    EXPECT_RANGE_EQ(result1, seqan3::views::minimiser(text1 | gapped_kmer_view, text1 | rev_gapped_kmer_view, 5));
+    EXPECT_RANGE_EQ(result1, text1 | gapped_kmer_view | minimiser_no_rev_view);
+    EXPECT_THROW(text1_short | gapped_kmer_view | minimiser_view1, std::invalid_argument);
+    auto empty_view = seqan3::views::minimiser(too_short_text | gapped_kmer_view,
+                                               too_short_text | rev_gapped_kmer_view,
+                                               5);
     EXPECT_TRUE(std::ranges::empty(empty_view));
-    EXPECT_RANGE_EQ(result3_gapped_no_rev, text3 | gapped_kmer_view | minimiser_view);
+    auto empty_view2 = too_short_text | gapped_kmer_view | minimiser_no_rev_view;
+    EXPECT_TRUE(std::ranges::empty(empty_view2));
+    EXPECT_RANGE_EQ(result3_gapped, seqan3::views::minimiser(text3 | gapped_kmer_view, text3 | rev_gapped_kmer_view, 5));
+    EXPECT_RANGE_EQ(result3_gapped_no_rev, text3 | gapped_kmer_view | minimiser_no_rev_view);
 }
 
 TEST_F(minimiser_test, window_too_big)
 {
     EXPECT_RANGE_EQ(result1_short, text1 | kmer_view | seqan3::views::minimiser(20));
     EXPECT_RANGE_EQ(result1_short, text1 | gapped_kmer_view | seqan3::views::minimiser(20));
+    EXPECT_RANGE_EQ(result1_short, seqan3::views::minimiser(text1 | kmer_view, text1 | rev_kmer_view, 20));
+    EXPECT_RANGE_EQ(result1_short, seqan3::views::minimiser(text1 | gapped_kmer_view, text1 | rev_gapped_kmer_view, 20));
 }
 
 TEST_F(minimiser_test, combinability)
 {
     auto stop_at_t = seqan3::views::take_until([] (seqan3::dna4 const x) { return x == 'T'_dna4; });
-    EXPECT_RANGE_EQ(result3_ungapped_no_rev_stop, text3 | stop_at_t | kmer_view | minimiser_view);
-    EXPECT_RANGE_EQ(result3_gapped_no_rev_stop, text3 | stop_at_t | gapped_kmer_view | minimiser_view);
+    EXPECT_RANGE_EQ(result3_ungapped_stop, text3 | stop_at_t | kmer_view | minimiser_no_rev_view);
+    EXPECT_RANGE_EQ(result3_gapped_stop, text3 | stop_at_t | gapped_kmer_view | minimiser_no_rev_view);
+
+#if SEQAN3_WORKAROUND_ISSUE_1754
```

Comment:
The same here. The reference issue seems to be closed/fixed?

Reply:
Now, this is now depending on #1953